doc/source/configuration/kayobe-automation.rst

```diff
@@ -0,0 +1,126 @@
+=================
+Kayobe Automation
+=================
+
```

Comment:
I would start with a very high level list of the components of the CI/CD system for context:

* Platform-specific CI/CD workflow definitions
* Platform-specific CD/CD runners
* Platform-independent Kayobe automation tooling

Reply:
Then you could list the supported platforms.

Reply:
Not sure if what I have added is what you are after.